Looks like a test needs to be updated with `six.text_type`
```
FAIL: corehq.apps.api.tests:TestReportPillow.test_xformPillowTransform
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/api/tests.py", line 992, in test_xformPillowTransform
    self.assertTrue(isinstance(cleaned['form']['meta']['appVersion'], str))
AssertionError: False is not true
```

--- a/corehq/apps/api/tests.py
+++ b/corehq/apps/api/tests.py
@@ -2,6 +2,7 @@ from __future__ import absolute_import
 from __future__ import unicode_literals
 import base64
 import json
+import six
 import uuid
 from collections import OrderedDict
 from copy import deepcopy
@@ -989,7 +990,7 @@ class TestReportPillow(TestCase):
         for fn in transform_functions:
             cleaned = fn(bad_appVersion)
             self.assertFalse(isinstance(cleaned['form']['meta']['appVersion'], dict))
-            self.assertTrue(isinstance(cleaned['form']['meta']['appVersion'], str))
+            self.assertTrue(isinstance(cleaned['form']['meta']['appVersion'], six.text_type))
             self.assertTrue(cleaned['form']['meta']['appVersion'], "CCODK:\"2.5.1\"(11126). v236 CC2.5b[11126] on April-15-2013")
 
 

